nym-vpn-core/crates/nym-vpn-account-controller/Cargo.toml

```diff
@@ -41,7 +41,7 @@ thiserror.workspace = true
 time.workspace = true
 tokio-util.workspace = true
 tokio-stream.workspace = true
-tokio = { workspace = true, features = ["fs"] }
+tokio = { workspace = true, features = ["fs", "rt-multi-thread", "macros"] }
```

Comment:
nit: those features are only requires for tests, thus should be put under `[dev-dependencies]`